doc/develop/getting_started/index.rst

```diff
@@ -386,12 +386,61 @@ contains toolchains for each of Zephyr's supported architectures, which
 include a compiler, assembler, linker and other programs required to build
 Zephyr applications.
 
-It also contains additional host tools, such as custom QEMU and OpenOCD builds
+For Linux, it also contains additional host tools, such as custom QEMU and OpenOCD builds
 that are used to emulate, flash and debug Zephyr applications.
 
-.. include:: ../toolchains/zephyr_sdk.rst
```

Comment:
I haven't looked at this file yet, but are there any remaining references to it?

Reply:
Just revisited. It still has a fair bit of useful info.